formats/log/note.go

```diff
@@ -29,27 +36,30 @@ import (
 // checkpoint is returned. The underlying note is always returned where possible.
 // The signatures on the note will include the log signature if no error is returned,
 // plus any signatures from otherVerifiers that were found.
-func ParseCheckpoint(chkpt []byte, origin string, logVerifier note.Verifier, otherVerifiers ...note.Verifier) (*Checkpoint, *note.Note, error) {
+func ParseCheckpoint(chkpt []byte, origin string, logVerifier note.Verifier, otherVerifiers ...note.Verifier) (*Checkpoint, *CheckpointNote, error) {
```

Comment:
Did you consider augmenting `CheckpointNote` struct to also contain the `*Checkpoint` and just return that?

Reply:
I've pushed a new commit that does this. I have mixed feelings. Let me know what you think.